### ISSUE TITLE
TestPilot projects not showing in FF 55 MacOS

### ISSUE THREAD

**Issue by toyg** [mozilla/testpilot#2749]
Whenever I click on any project on testpilot.firefox.com, the page just keeps loading indefinitely. The console gets continuously spammed with the following message in an endless loop:

```Sending message that cannot be cloned. Are you trying to send an XPCOM object? addonManager.js:206:10```

Firefox 55.0 on MacOSX 10.12.6. I tried in Safe Mode too.

About:support raw data:
{
  "application": {
    "name": "Firefox",
    "osVersion": "Darwin 16.7.0",
    "version": "55.0",
    "buildID": "20170803105720",
    "userAgent": "Mozilla/5.0 (Macintosh; Intel Mac OS X 10.12; rv:55.0) Gecko/20100101 Firefox/55.0",
    "safeMode": true,
    "updateChannel": "release",
    "supportURL": "https://support.mozilla.org/1/firefox/55.0/Darwin/en-GB/",
    "numTotalWindows": 1,
    "numRemoteWindows": 1,
    "remoteAutoStart": true,
    "currentContentProcesses": 3,
    "maxContentProcesses": 4,
    "autoStartStatus": 0,
    "keyGoogleFound": true,
    "keyMozillaFound": true
  },
  "modifiedPreferences": {
    "accessibility.typeaheadfind.flashBar": 0,
    "browser.cache.disk.filesystem_reported": 1,
    "browser.cache.frecency_experiment": 2,
    "browser.cache.disk.smart_size.first_run": false,
    "browser.cache.disk.smart_size.use_old_max": false,
    "browser.cache.disk.capacity": 358400,
    "browser.places.smartBookmarksVersion": 8,
    "browser.search.useDBForOrder": true,
    "browser.sessionstore.upgradeBackup.latestBuildID": "20170803105720",
    "browser.startup.homepage_override.mstone": "55.0",
    "browser.startup.homepage_override.buildID": "20170803105720",
    "browser.tabs.warnOnClose": false,
    "browser.tabs.remote.force-enable": true,
    "browser.tabs.remote.autostart.2": true,
    "browser.urlbar.lastSuggestionsPromptDate": 20170808,
    "browser.urlbar.timesBeforeHidingSuggestionsHint": 0,
    "browser.urlbar.daysBeforeHidingSuggestionsPrompt": 0,
    "dom.push.userAgentID": "a3cb8fc4e7a643fba9c59b3182d258fe",
    "dom.ipc.processCount": 4,
    "extensions.lastAppVersion": "55.0",
    "font.internaluseonly.changed": true,
    "media.webrtc.debug.log_file": "/private/var/folders/wc/csdscxjx55n5wkft51130s2h0000gn/T/TemporaryItems/WebRTC.log",
    "media.gmp-widevinecdm.lastUpdate": 1493044541,
    "media.gmp-gmpopenh264.lastUpdate": 1493044539,
    "media.gmp-gmpopenh264.abi": "x86_64-gcc3",
    "media.gmp-manager.lastCheck": 1502219921,
    "media.gmp-widevinecdm.abi": "x86_64-gcc3",
    "media.gmp-widevinecdm.version": "1.4.8.903",
    "media.benchmark.vp9.versioncheck": 2,
    "media.gmp.storage.version.observed": 1,
    "media.benchmark.vp9.fps": 129,
    "media.gmp-gmpopenh264.version": "1.6",
    "media.gmp-manager.buildID": "20170803105720",
    "network.prefetch-next": false,
    "network.dns.disablePrefetch": true,
    "network.http.speculative-parallel-limit": 0,
    "network.predictor.cleaned-up": true,
    "network.cookie.prefsMigrated": true,
    "places.history.expiration.transient_current_max_pages": 112348,
    "places.database.lastMaintenance": 1502106347,
    "plugin.disable_full_page_plugin_for_types": "application/pdf",
    "plugins.ctprollout.cohortSample": "0.300813",
    "plugins.ctprollout.cohort": "excluded",
    "security.sandbox.content.tempDirSuffix": "{52012051-0a57-8444-bfd7-7e1d01ad6d84}",
    "services.sync.declinedEngines": "adblockplus",
    "services.sync.lastPing": 1502206281,
    "services.sync.lastSync": "Wed Aug 09 2017 02:43:56 GMT+0100 (BST)",
    "services.sync.numClients": 3,
    "services.sync.engine.greasemonkey": true,
    "services.sync.engine.prefs.modified": false,
    "storage.vacuum.last.index": 1,
    "storage.vacuum.last.places.sqlite": 1500132111,
    "storage.vacuum.last.queue.sqlite": 1502206466
  },
  "lockedPreferences": {},
  "javaScript": {
    "incrementalGCEnabled": true
  },
  "accessibility": {
    "isActive": false,
    "forceDisabled": 0
  },
  "libraryVersions": {
    "NSPR": {
      "minVersion": "4.15",
      "version": "4.15"
    },
    "NSS": {
      "minVersion": "3.31",
      "version": "3.31"
    },
    "NSSUTIL": {
      "minVersion": "3.31",
      "version": "3.31"
    },
    "NSSSSL": {
      "minVersion": "3.31",
      "version": "3.31"
    },
    "NSSSMIME": {
      "minVersion": "3.31",
      "version": "3.31"
    }
  },
  "userJS": {
    "exists": false
  },
  "crashes": {
    "submitted": [],
    "pending": 0
  },
  "sandbox": {
    "contentSandboxLevel": 1,
    "effectiveContentSandboxLevel": 1
  },
  "graphics": {
    "numTotalWindows": 2,
    "numAcceleratedWindows": 0,
    "windowLayerManagerType": "Basic",
    "windowLayerManagerRemote": true,
    "currentAudioBackend": "audiounit",
    "numAcceleratedWindowsMessage": [
      ""
    ],
    "adapterDescription": "",
    "adapterVendorID": "0x1002",
    "adapterDeviceID": "0x67ef",
    "adapterRAM": "",
    "adapterDrivers": "",
    "driverVersion": "",
    "driverDate": "",
    "webgl1Renderer": "WebGL is currently disabled.",
    "webgl1Version": "-",
    "webgl1DriverExtensions": "-",
    "webgl1Extensions": "-",
    "webgl1WSIInfo": "-",
    "webgl2Renderer": "WebGL is currently disabled.",
    "webgl2Version": "-",
    "webgl2DriverExtensions": "-",
    "webgl2Extensions": "-",
    "webgl2WSIInfo": "-",
    "info": {
      "AzureCanvasBackend": "skia",
      "AzureFallbackCanvasBackend": "none",
      "AzureContentBackend": "skia",
      "AzureCanvasAccelerated": 0,
      "ApzWheelInput": 1,
      "ApzDragInput": 1,
      "TileHeight": 1024,
      "TileWidth": 1024
    },
    "featureLog": {
      "features": [
        {
          "name": "HW_COMPOSITING",
          "description": "Compositing",
          "status": "blocked",
          "log": [
            {
              "type": "default",
              "status": "available"
            },
            {
              "type": "runtime",
              "status": "blocked",
              "message": "Acceleration blocked by safe-mode"
            }
          ]
        },
        {
          "name": "OPENGL_COMPOSITING",
          "description": "OpenGL Compositing",
          "status": "unavailable",
          "log": [
            {
              "type": "default",
              "status": "unavailable",
              "message": "Hardware compositing is disabled"
            }
          ]
        },
        {
          "name": "WEBRENDER",
          "description": "WebRender",
          "status": "unavailable",
          "log": [
            {
              "type": "default",
              "status": "opt-in",
              "message": "WebRender is an opt-in feature"
            },
            {
              "type": "runtime",
              "status": "unavailable",
              "message": "Build doesn't include WebRender"
            }
          ]
        }
      ],
      "fallbacks": []
    },
    "crashGuards": []
  },
  "experiments": [],
  "extensions": [
    {
      "name": "Add-on Compatibility Reporter",
      "version": "2.2.4",
      "isActive": false,
      "id": "compatibility@addons.mozilla.org"
    },
    {
      "name": "bitwarden - Free Password Manager",
      "version": "1.14.3",
      "isActive": false,
      "id": "{446900e4-71c2-419f-a6a7-df9c091e268b}"
    },
    {
      "name": "Browse Images",
      "version": "0.4.2.1-signed.1-signed",
      "isActive": false,
      "id": "{62b958b4-9962-4fc2-9983-01a9a42d6f2d}"
    },
    {
      "name": "DataVault Toolbar",
      "version": "5.1.4.1-signed",
      "isActive": false,
      "id": "datavault@ascendo.inc"
    },
    {
      "name": "Disconnect",
      "version": "5.18.21",
      "isActive": false,
      "id": "2.0@disconnect.me"
    },
    {
      "name": "DownThemAll!",
      "version": "3.0.8",
      "isActive": false,
      "id": "{DDC359D1-844A-42a7-9AA1-88A850A938A8}"
    },
    {
      "name": "Firebug",
      "version": "2.0.19",
      "isActive": false,
      "id": "firebug@software.joehewitt.com"
    },
    {
      "name": "Greasemonkey",
      "version": "3.11",
      "isActive": false,
      "id": "{e4a8a97b-f2ed-450b-b12d-ee082ba24781}"
    },
    {
      "name": "Hacker News Reader",
      "version": "1.7",
      "isActive": false,
      "id": "hnreader@pragmatictester.com"
    },
    {
      "name": "HN Utility Suite",
      "version": "initial.rev20.1.1-signed.1-signed",
      "isActive": false,
      "id": "jid0-rhGAMphPt2NGrV7gjxnbEERdBBE@jetpack"
    },
    {
      "name": "img2tab",
      "version": "1.67.1-signed.1-signed",
      "isActive": false,
      "id": "sumeetkpatel@gmail.com"
    },
    {
      "name": "It's All Text!",
      "version": "1.9.3",
      "isActive": false,
      "id": "itsalltext@docwhat.gerf.org"
    },
    {
      "name": "JavaScript Debugger",
      "version": "0.9.89.1-signed.1-signed",
      "isActive": false,
      "id": "{f13b157f-b174-47e7-a34d-4815ddfdfeb8}"
    },
    {
      "name": "JSONovich",
      "version": "1.99.4",
      "isActive": false,
      "id": "jsonovich@lackoftalent.org"
    },
    {
      "name": "JSONView",
      "version": "1.2.4",
      "isActive": false,
      "id": "jsonview@brh.numbera.com"
    },
    {
      "name": "LastPass: Free Password Manager",
      "version": "4.1.54",
      "isActive": false,
      "id": "support@lastpass.com"
    },
    {
      "name": "Live HTTP headers",
      "version": "0.17.1-signed.1-signed",
      "isActive": false,
      "id": "{8f8fe09b-0bd3-4470-bc1b-8cad42b8203a}"
    },
    {
      "name": "MeasureIt",
      "version": "0.4.15",
      "isActive": false,
      "id": "{75CEEE46-9B64-46f8-94BF-54012DE155F0}"
    },
    {
      "name": "New Tab Tools",
      "version": "81",
      "isActive": false,
      "id": "newtabtools@darktrojan.net"
    },
    {
      "name": "Old Lion Fullscreen",
      "version": "1.0.1-signed.1-signed",
      "isActive": false,
      "id": "old-lion-fullscreen@flyingtealeaf.com"
    },
    {
      "name": "OmniSidebar",
      "version": "1.6.15",
      "isActive": false,
      "id": "osb@quicksaver"
    },
    {
      "name": "Pinboard Extension",
      "version": "1.2.1-signed.1-signed",
      "isActive": false,
      "id": "pinboardff@pinboard.in"
    },
    {
      "name": "Reddit Enhancement Suite",
      "version": "5.8.4",
      "isActive": false,
      "id": "jid1-xUfzOsOFlzSOXg@jetpack"
    },
    {
      "name": "Remote XUL Manager",
      "version": "1.3.2",
      "isActive": false,
      "id": "remotexulmanager@xulforge.com"
    },
    {
      "name": "REST Easy",
      "version": "0.3.1.1-signed",
      "isActive": false,
      "id": "rest-easy@quickmediasolutions.com"
    },
    {
      "name": "RESTClient",
      "version": "2.0.5",
      "isActive": false,
      "id": "{ad0d925d-88f8-47f1-85ea-8463569e756e}"
    },
    {
      "name": "Roomy Bookmarks Toolbar",
      "version": "1.4.14",
      "isActive": false,
      "id": "ALone-live@ya.ru"
    },
    {
      "name": "SSleuth",
      "version": "0.5.3",
      "isActive": false,
      "id": "ssleuth@github"
    },
    {
      "name": "Stylish",
      "version": "2.0.7",
      "isActive": false,
      "id": "{46551EC9-40F0-4e47-8E18-8E5CF550CFB8}"
    },
    {
      "name": "TabTrekker",
      "version": "1.2.3",
      "isActive": false,
      "id": "jid1-AahxeYITR16k5Q@jetpack"
    },
    {
      "name": "Test Pilot",
      "version": "2.0.2-dev-0d66c35",
      "isActive": false,
      "id": "@testpilot-addon"
    },
    {
      "name": "Thumbnail Zoom",
      "version": "1.4.3.1-signed.1-signed",
      "isActive": false,
      "id": "{E10A6337-382E-4FE6-96DE-936ADC34DD04}"
    },
    {
      "name": "Thumbnail Zoom Plus",
      "version": "4.2",
      "isActive": false,
      "id": "thumbnailZoom@dadler.github.com"
    },
    {
      "name": "TinEye Reverse Image Search",
      "version": "1.3.2",
      "isActive": false,
      "id": "tineye@ideeinc.com"
    },
    {
      "name": "uBlock Origin",
      "version": "1.13.8",
      "isActive": false,
      "id": "uBlock0@raymondhill.net"
    },
    {
      "name": "Wayback Machine",
      "version": "1.8.1",
      "isActive": false,
      "id": "wayback_machine@mozilla.org"
    },
    {
      "name": "Web Developer",
      "version": "1.2.13",
      "isActive": false,
      "id": "{c45c406e-ab73-11d8-be73-000a95be3b12}"
    },
    {
      "name": "XPath Checker",
      "version": "0.4.4.1-signed.1-signed",
      "isActive": false,
      "id": "{7eb3f691-25b4-4a85-9038-9e57e2bcd537}"
    }
  ],
  "features": [
    {
      "name": "Application Update Service Helper",
      "version": "2.0",
      "id": "aushelper@mozilla.org"
    },
    {
      "name": "Click-to-Play staged rollout",
      "version": "1.2",
      "id": "clicktoplay-rollout@mozilla.org"
    },
    {
      "name": "Firefox Screenshots",
      "version": "10.10.0",
      "id": "screenshots@mozilla.org"
    },
    {
      "name": "Follow-on Search Telemetry",
      "version": "0.9.1",
      "id": "followonsearch@mozilla.com"
    },
    {
      "name": "Multi-process staged rollout",
      "version": "2.0",
      "id": "e10srollout@mozilla.org"
    },
    {
      "name": "Pocket",
      "version": "1.0.5",
      "id": "firefox@getpocket.com"
    },
    {
      "name": "Shield Recipe Client",
      "version": "55.1",
      "id": "shield-recipe-client@mozilla.org"
    },
    {
      "name": "Web Compat",
      "version": "1.1",
      "id": "webcompat@mozilla.org"
    }
  ]
}

**Comment by lmorchard** [comment]
Maybe a duplicate of #2746 ?

**Comment by toyg** [comment]
Might well be.

**Comment by SoftVision-PaulOiegas** [comment]
@toyg If you haven't seen the workaround in #2746, please try and add in "Menu > Options > Content > Language" displayed pop up, the English US [en-US] language at the bottom of the stack. 
This is just for a while until we merge the fix for the issue.

I am closing this one in favor of the above mentioned one since they seem related. Please follow the thread there.

**Comment by toyg** [comment]
Yeah that works. Nice!
